Jenkinsfile

```diff
@@ -219,14 +222,17 @@ try {
 					milestone(label: 'stage_deploy_test_start')
 
 					container('bfd-cbc-build') {
+						// Unset any pre-existing session variables
 						// Assume new role session for each deploy to prevent timeout
-						withCredentials([string(credentialsId: 'bfd-aws-assume-role', variable: 'awsAssumeRole')]) {
-							awsCredentials = sh(returnStdout: true, script: "aws sts assume-role --role-arn ${awsAssumeRole} --role-session-name session --output text --query Credentials").trim().split(/\s+/)
-							env.AWS_DEFAULT_REGION = 'us-east-1'
-							env.AWS_ACCESS_KEY_ID = awsCredentials[0]
-							env.AWS_SECRET_ACCESS_KEY = awsCredentials[2]
-							env.AWS_SESSION_TOKEN = awsCredentials[3]
-						}
+                        withEnv(['AWS_ACCESS_KEY_ID=','AWS_SECRET_ACCESS_KEY=','AWS_SESSION_TOKEN=']) {
+                            withCredentials([string(credentialsId: 'bfd-aws-assume-role', variable: 'awsAssumeRole')]) {
+								awsCredentials = sh(returnStdout: true, script: 'aws sts assume-role --role-arn "$awsAssumeRole" --role-session-name bfd-multibranch-and-multistage-pipeline --output text --query Credentials').trim().split(/\s+/)
+                                env.AWS_DEFAULT_REGION = 'us-east-1'
+                                env.AWS_ACCESS_KEY_ID = awsCredentials[0]
+                                env.AWS_SECRET_ACCESS_KEY = awsCredentials[2]
+                                env.AWS_SESSION_TOKEN = awsCredentials[3]
+                            }
+                        }
```

Comment:
we lost the proper indentation on this block

Reply:
Yikes. Let me address that. Thanks!